stitch/src/main/java/com/mongodb/stitch/android/StitchClient.java

```diff
@@ -663,11 +659,11 @@ String getAuthProvidersLoginRoute(String providerType) {
 
         private final String FUNCTIONS = String.format("app/%s/functions/call", _clientAppId);
         private final String PUSH = String.format("app/%s/push/providers", _clientAppId);
-        private final String USERPASS_REGISTER = "/register";
-        private final String USERPASS_CONFIRM = "local/userpass/confirm";
-        private final String USERPASS_CONFIRM_SEND = "local/userpass/confirm/send";
-        private final String USERPASS_RESET = "local/userpass/reset";
-        private final String USERPASS_RESET_SEND = "local/userpass/reset/send";
+        private final String USERPASS_REGISTER = "register";
+        private final String USERPASS_CONFIRM = "local-userpass/confirm";
```

Comment:
[nit] shouldn't we just reuse the same common string and `String.format` them here or `+`?

Reply:
We should.